modules/itsi/module.json

```diff
@@ -20,7 +20,7 @@
 			"twitter": "@ForgeVTT"
 		}
 	],
-	"url": "https://www.dndbeyond.com/sources/itsi",
+	"url": "https://www.dndbeyond.com/sources/dnd/itsi",
```

Comment:
Redirects to `https://www.dndbeyond.com/sources/dnd/itsi/intro-to-stormwreck-isle`